public/app/features/dashboard-scene/scene/RowRepeaterBehavior.ts

```diff
@@ -99,10 +99,18 @@ export class RowRepeaterBehavior extends SceneObjectBase<RowRepeaterBehaviorStat
 
     let maxYOfRows = 0;
 
+    const emptyVariablePlaceholderOption = {
+      values: ['placeholder'],
```

Comment:
@axelavargas 
not sure "placeholder" is a good emtpy value, maybe an empty string is better?